src/loader.c

```diff
@@ -172,6 +173,10 @@ int main(void)
 {
     hal_init();
     spi_flash_probe();
+#ifdef UART_FLASH
+    uart_init((4 * 115200), 8, 'N', 1);
```

Comment:
What is `4 *` here? This loader.c seems like it should be generic and just pass 115200 and the porting implementation should handle the `4 *`?